store.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap" rel="stylesheet">
+    <link href="/Users/mayrarakhimbayeva/Desktop/new demo/HTML-CSS-Challenges/Form-Controls/HTML-CSS-Coursework-Week2/css/css style for storehtml/storestyle.css">
+</head>
+<body>
+    <header class="header">
+       
+        <a href="#">
+            <div class="header__logo">
+                
+              <img src="img/karma-logo.svg"  alt="logo of karma">
+                
+            </div>
+        </a>
+        
+        <section class="navigation_list">
+            <nav class="navbar"> 
+                <!-- add space between items-->
+                <a class="navigation_item" href="#"> <strong>Meet Karma </strong> </a>
+                <a class="navigation_item" href="#"> How it Works </a>
+                <a class="navigation_item" href="#"> Store </a>
+                <a class="navigation_item" href="#"> Blog </a>
+                <a class="navigation_item" href="#"> Help </a>
+                <a class="navigation_item" href="#"> Login </a>
+             </nav>
+        </section> 
+
+   </header> 
+
+
+    <main>
+
+    <section class="content">
+
+       <section class="rightcontent">
+             
+                <img src="img/secondbackground.jpg" alt="another girl with lap top and coffee" class="secondpic">
+            
+        </section>
+
+        <section class="leftcontent"> 
+            <div class="title">
+                <h2>Order your Karma wifi device today!</h2>
+            </div>
+
+            <form action="">
+              <fieldset class="fieldset"> 
+                  <div class="details" >
+                        <div class="input_box">
+                        <label for="fname">First name<span> *</span></label> <br>
+                        <input type="text" id="fname" name="fname"><br>
+                        </div>
+                        <br>
+
+                        <div class="input_box">
+                            <label for="lname"> Last name<span> *</span></label> <br>
+                            <input type="text" id="lname" name="lname">
```

Comment:
Add the `required` attribiute to the `input`s with `*` on their label. Browsers avoid submitting the form if those required fields are empty.